samples/net/azure_iot_hub/boards/native_sim.conf

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General
+CONFIG_LOG_BUFFER_SIZE=2048
+CONFIG_HW_ID_LIBRARY=y
+CONFIG_ASSERT=y
+CONFIG_FPU=y
+
+# Heap and stacks
+CONFIG_INIT_STACKS=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+
+# Networking config
+CONFIG_NET_IPV6=y
+CONFIG_NET_TCP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_PKT_RX_COUNT=500
+CONFIG_NET_PKT_TX_COUNT=500
+CONFIG_NET_BUF_RX_COUNT=600
+CONFIG_NET_BUF_TX_COUNT=600
```

Comment:
Why are these values very different from the ones in the other samples?

Reply:
just mirroring the azure iot qemu_x86.conf, but no real reason. Should we set 32 to all?